requirements.txt

```diff
@@ -4,6 +4,9 @@
 #
 # If you have problems running this, try the util/venv-builder.sh script.
 
+ansible
+markupsafe
+
 # Core with Azure dependencies
 ansible[azure]
```

Comment:
We had problems installing both `ansible` and `ansible[azure]`. Although it looks correct, we need to take that out. :-(

Reply:
...and maybe we should add that as a comment, since it does look odd not to specify things the normal way.

Reply:
Removing `ansible` and leaving `ansible[azure]` should work. I think that changing `ansible[azure]` to just `azure` should do the trick as well, though I haven't tested it.

Reply:
What kind of problems did you face? This worked for me flawlessly. And because it is Docker, then it should always work the same way.

Reply:
@hammady IIRC, I faced problems trying to provision to Azure when installing `ansible` followed by `ansible[azure]`

Reply:
It's possible it's fixed now. I'll take a look.